packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -131,7 +131,11 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/cha
      {{- end }}
      imagePullPolicy: {{ .Values.initChownData.image.pullPolicy }}
      securityContext:
-@@ -41,9 +40,9 @@
++      runAsNonRoot: false
```

Comment:
Weird, wouldn't runAsUser handle this for you ? Or do you need both to enable root user.

Reply:
`runAsNonRoot` will force you to run as non root. Due to the CIS run we made all the pod run with `runAsNonRoot=true`. This initChown container needs to run as root user so that `runAsNonRoot=true` can not be set. That's why we can set it on container level so it is overridden.

Reply:
Mmk, so you need both then.